src/Docker/Runner/WorkingDirectory.php

```diff
@@ -135,17 +123,12 @@ public function moveOutputToInput()
         );
 
         // create empty output
-        $fs = new Filesystem();
-        $fs->mkdir($this->workingDir);
-
-        $structure = [
+        $fs->mkdir([
             $this->workingDir . "/tmp",
             $this->workingDir . "/data/out",
```

Comment:
If mkdir is recursive then maybe we don't need this one

Reply:
je to tak